pacts/pact-consumer-api-pact-provider-api.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
This is stored in a shared `pacts` directory separate from the Provider and Consumer for now. The PACTs should be stored in some shared service, whether this be S3 or the PACT broker.